src/doc-editors/CustomResponsesEditor.vue

```diff
@@ -63,20 +63,37 @@
         <div v-if="['monitor', 'custom'].includes(localDoc.type)"
              class="column is-8">
           <div class="custom-panel">
+            <div v-if="localDoc.type === 'custom'"
+                 class="field">
+              <label class="label is-small status-code-label">
+                Status code
+              </label>
+              <div class="columns mb-0">
+                <div class="column is-5 pb-0">
```

Comment:
Change to `is-3` to match the Header Key input width
```suggestion
                <div class="column is-3 pb-0">
```

Reply:
Done